index.bs

```diff
@@ -3116,11 +3216,15 @@ dictionary MLLogicalNotSupportLimits {
 
 partial dictionary MLOpSupportLimits {
   MLBinarySupportLimits equal;
+  MLBinarySupportLimits notEqual;
   MLBinarySupportLimits greater;
   MLBinarySupportLimits greaterOrEqual;
   MLBinarySupportLimits lesser;
   MLBinarySupportLimits lesserOrEqual;
   MLLogicalNotSupportLimits logicalNot;
+  MLLogicalNotSupportLimits logicalAnd;
+  MLLogicalNotSupportLimits logicalOr;
+  MLLogicalNotSupportLimits logicalXor;
 };
 </script>
 
```

Comment:
Need to add the new ops to the `<div dfn-for...` just below this line.

This ensures the arguments in the IDL get linked up appropriately.